geonode/people/models.py

```diff
@@ -246,6 +246,7 @@ def _notify_account_activated(self):
                     "site_name": current_site.name,
                     "email": self.email,
                     "inviter": self,
+                    'LOGIN_URL': settings.LOGIN_URL,
```

Comment:
I think the Circle CI test does not like this to have `'` instead of `"` for the variable `LOGIN_URL` ? See failed test.

Reply:
> ^@^@would reformat /usr/src/geonode/geonode/people/models.py
>Oh no! 💥 💔 💥
1 file would be reformatted, 479 files would be left unchanged.